### PR TITLE
Feature/sergeyro/css modules migration 9

### PR DESCRIPTION
**Pull request by SergeyRoyt**
- [ ] 1. AlertBanner
- [ ] 2. AlertBannerButton
- [ ] 3. AlertBannerLink
- [ ] 4. AlertBannerText

### CODE REVIEW

**Comment by SergeyRoyt** [comment]
TODO in `AlertBannerButton.tsx` - styles seems so not load - seen in snapshots - problems with ts + css modules

File: src/utils/test-utils.js [hunk 1]
```diff
@@ -12,8 +12,7 @@ export const ELEMENT_TYPES = {
   COLOR_PICKER_ITEM: "color-picker-item",
   ICON_BUTTON: "icon-button",
   CHIP: "chip",
-  TAB: "tab",
-  TAB_PANEL: "tab-panel",
```
Comment (3dyonic) [inline src/utils/test-utils.js:16]: why did you remove them?

**Review by 3dyonic — COMMENTED** [review]
please see my remarks

File: src/components/AlertBanner/AlertBannerButton/AlertBannerButton.module.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@import "../../../styles/global-css-settings";
+@import "../../../styles/typography";
+@import "../../../styles/themes";
+
+:global(.monday-style-button).button {
+  -webkit-font-smoothing: auto;
```
[the remark this answers is not in the document]
Comment (SergeyRoyt) [reply src/components/AlertBanner/AlertBannerButton/AlertBannerButton.module.scss:6]: Do u think it's better to fix it here or in separate PR?

File: src/components/AlertBanner/AlertBanner.jsx [hunk 3]
```diff
@@ -1,18 +1,38 @@
...
+    },
+    ref
+  ) => {
     const classNames = useMemo(() => {
-      return cx(className, "monday-alert-banner", `monday-alert-banner--background-color-${backgroundColor}`);
+      return cx(
+        className,
+        styles.banner,
+        "monday-alert-banner",
+        styles[camelCase("background-color-" + backgroundColor)],
```
[the remark this answers is not in the document]
Comment (SergeyRoyt) [reply src/components/AlertBanner/AlertBanner.jsx]: We can live without it - removed

**Review by 3dyonic — APPROVED** [review]

File: src/utils/test-utils.js (hunk 1 above)
Comment (SergeyRoyt) [reply src/utils/test-utils.js:16]: believe me - it's not :)

File: src/components/AlertBanner/AlertBannerButton/AlertBannerButton.module.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (SergeyRoyt) [reply src/components/AlertBanner/AlertBannerButton/AlertBannerButton.module.scss]: Did it, but not sure if it solved the bug - cause I'm not aware what is this bug about
@3dyonic please check if it solved anything

**Comment by SergeyRoyt** [comment]
Merged into [Vibe2.0](https://github.com/mondaycom/monday-ui-react-core/pull/885)
